src/main/scala/sigmastate/serialization/SigmaSerializer.scala

```diff
@@ -15,9 +15,14 @@ trait Serializer[TFamily, T <: TFamily] {
     parseBody(bytes, 0)._1
   }
 
-  def parseBody(bytes: Array[Byte], pos: Position): (TFamily, Consumed)
+  // todo remove after transition to new API is complete
+  def parseBody(bytes: Array[Byte], pos: Position): (TFamily, Consumed) = ???
   def serializeBody(obj: T): Array[Byte] = toBytes(obj)
 
+  // todo remove "not implemented" after implemented everywhere
+  def parseBody(r: ByteReader): TFamily = ???
```

Comment:
@ergomorphic Check out the updated `parseBode` and `serializeBody` signatures for passing reader/writer. Does it sound good?